app/controllers/hello_react_controller.rb

```diff
@@ -1,9 +1,9 @@
 class HelloReactController < ApplicationController
-  def contracts
-    @contractor = Contractor.new("seeds/offers.json")
-    @contractor.build_contracts
-    render json: @contractor.get_parsed
-  end
+  # def contracts
+  #   @contractor = Contractor.new("seeds/offers.json")
+  #   @contractor.build_contracts
+  #   render json: @contractor.get_parsed
+  # end
 end
```

Comment:
If we don't need this code, let's delete it (and extract things below into somewhere appropriate).